### PR TITLE
toast

### PR DESCRIPTION
**Pull request by RaghavendraaReddy**
- review
- [Live CodeSandBox](https://6p4deh.csb.app/toast.html)

### CODE REVIEW

**Review by Rajitha04 — APPROVED** [review]
Screenshot_2022-02-21-20-31-49-97_40deb401b9ffe8e1df2f1cc5ba480b12
Try to use rem instead of px when the value is greater in pixels,  also try to add some more padding for better view

File: text.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    font-size: 55px;
+    margin-left: 45px;
+}
+.a{
+    justify-content: flex-end;
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+}
+.a1{
```
Comment (anupkrbid) [inline text.css]: naming can be better

File: text.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        <h2>Components</h2>
+        <div><a href="#">Alert</a></div>
+        <div><a href="#">Avatar</a></div>
+        <div><a href="#">Badge</a></div>
+        <div><a href="#">Button</a></div>
+        <div><a href="#">Card</a></div>
+        <div><a href="#">Grid</a></div>
+        <div><a href="#">Image</a></div>
+        <div><a href="#">Input</a></div>
+        <div><a href="#">Toast</a></div>
```
Comment (anupkrbid) [inline text.html]: use list instead of div in this case

File: text.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                <h1>Heading 1</h1>
+                <h2>Heading 2</h2>
+                <h3>Heading 3</h3>
+                <h4>Heading 4</h4>
+                <h5>Heading 5</h5>
+                <h6>Heading 6</h6>
+            </div>
+            <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=31px&dsblur=0px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=13.5px&lh=119%25&si=false&es=2x&wm=false&code=%2520%253Cdiv%2520class%253D%2522heading%2520mg%2522%253E%250A%2520%2520%2520%2520%253Ch1%253EHeading%25201%253C%252Fh1%253E%250A%2520%2520%2520%2520%253Ch2%253EHeading%25202%253C%252Fh2%253E%250A%2520%2520%2520%2520%253Ch3%253EHeading%25203%253C%252Fh3%253E%250A%2520%2520%2520%2520%253Ch4%253EHeading%25204%253C%252Fh4%253E%250A%2520%2520%2520%2520%253Ch5%253EHeading%25205%253C%252Fh5%253E%250A%2520%2520%2520%2520%253Ch6%253EHeading%25206%253C%252Fh6%253E%250A%2520%253C%252Fdiv%253E"
+            style="width: 225px; height: 186px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (anupkrbid) [inline text.html]: avoid inline-style

**Review by anupkrbid — APPROVED** [review]

**Pull request merged**